src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinMinecraftServer.java

```diff
@@ -0,0 +1,19 @@
+package com.mitchej123.hodgepodge.mixins.early.minecraft;
+
+import net.minecraft.server.MinecraftServer;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.Constant;
+import org.spongepowered.asm.mixin.injection.ModifyConstant;
+
+import com.mitchej123.hodgepodge.config.TweaksConfig;
+
+@Mixin(MinecraftServer.class)
+public class MixinMinecraftServer {
```

Comment:
Can you rename this to something like `MixinMinecraftServer_AutoSaveInterval`

Also, this should be under `early/minecraft/server/`

Reply:
I wasnt sure about the server package as the mixin inside there also doesnt origin in the minecraft server package. Or am I missing the point of this package?